concent_api/test_server_script.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+from json import JSONDecodeError
+import datetime
+import random
+
+import requests
+
+
+DEFAULT_HEADERS = {
+    'Content-Type': 'application/json',
+    'concent-client-public-key': '85cZzVjahnRpUBwm0zlNnqTdYom1LF1P1WNShLg17cmhN2UssnPrCjHKTi5susO3wrr/q07eswumbL82b4HgOw==',
+}
+
+
+class InvalidOption(Exception):
+    pass
+
+
+def create_data(message_type):
+    current_time = int(datetime.datetime.now().timestamp())
+    data = {
+        "type":         message_type,
+        "timestamp":    current_time,
+        "message_task_to_compute": {
+            "type":         "MessageTaskToCompute",
+            "timestamp":    current_time,
+            "task_id":      5,
+            "deadline":     current_time + 6000
+        }
+    }
+    if message_type == "MessageRejectReportComputedTask":
```

Comment:
This works in a weird way. You should just create a separate function that returns a `MessageRejectReportComputedTask`.